### ISSUE TITLE
[Documentation] Inclusion within `crowdsec-docs`

### ISSUE THREAD

**Issue by LaurenceJJones** [maxlerebourg/crowdsec-bouncer-traefik-plugin#188]
Hey 👋🏻 

Over the weeks at @crowdsecurity we have been discussing our current documentation and as it stands we only host first party supported documentation. However, since this remediation is very popular we stand to both benefit from an inclusion of material based on this remediation and how it can be used.

Would you be happy for us to do this? we always want to seek permission before doing something as what our intention:

- Use the readme.md as a starting point
- Refactor the current format to be more aligned with our first party:
  - Heading (Reffer to your readme)
  - Getting started (simply remediation)
  - Getting started (AppSec)
  - Configuration options (this might get little complicated as dynamic configuration and labels are two separate)

It will be on us to make sure the documentation stays up to date with your current options. However, I have seen this is mostly stable and not much breaking changing / new configuration options have been included.

Let me know your thoughts!

**Comment by mathieuHa** [comment]
Hi @LaurenceJJones,

Yes, I think It's a good Idea.  Can you just clarify one point for us. We understood that we could help you write a first version of documentation with the structure you mentioned that would be published in Crowdsec doc website. Is that correct ? Like for instance adding a Traefik Remediation component documented like the https://docs.crowdsec.net/u/bouncers/haproxy HAProxy one

Please let us know if that what you meant

**Comment by LaurenceJJones** [comment]
> Hi @LaurenceJJones,
> 
> Yes, I think It's a good Idea. Can you just clarify one point for us. We understood that we could help you write a first version of documentation with the structure you mentioned that would be published in Crowdsec doc website. Is that correct ? Like for instance adding a Traefik Remediation component documented like the https://docs.crowdsec.net/u/bouncers/haproxy HAProxy one
> 
> Please let us know if that what you meant

It up to you, we are happy to do it or if you wish to collaborate together on the PR then we invite you too also. It would be like the HAProxy format, provides context at the top then some "classical" examples and then goes into the configuration structure.

**Comment by mathieuHa** [comment]
Yes I would like that, I'll create in the next following days a PR for this when I got the time
